ASM/src/coop_state.asm

```diff
@@ -23,6 +23,7 @@ MW_PROGRESSIVE_ITEMS_ENABLE:
 
 OUTGOING_KEY:
 .word 0x00000000
+.word 0x00000000
```

Comment:
To make it easier to support multiple versions of the co-op context at once, replace this with a 4-byte gap and move the `OUTGOING_KEY` symbol to the end of the co-op context. This preserves the relative addresses of the other symbols.